security-team/security-groups.md

```diff
@@ -0,0 +1,11 @@
+## The Security Team
+
+Owners:
+
+- [xuzhonghu@huawei.com](mailto:xuzhonghu@huawei.com)
+
+Members:
+
+- [xuzhonghu@huawei.com](mailto:xuzhonghu@huawei.com)
+- [xieqianglong@huawei.com](mailto:xieqianglong@huawei.com)
+- [lizhencheng6@huawei.com](mailto:lizhencheng6@huawei.com)
```

Comment:
please donot use personal email, will apply a email group

Reply:
After applying for an email group, the content will be updated.

Reply:
Security team is a team consists of different people to handle security issues.
I think it's fine to list team members here, but please make sure people report security issues to a email group.